auth/src/main/res/values/styles.xml

```diff
@@ -31,33 +31,6 @@
         <item name="android:fontFamily">sans-serif</item>
     </style>
 
-    <style name="FirebaseUI.Text.T01">
```

Comment:
I also had to rebuild the `baseline` for some reason. It was probably something on my machine, but it cleaned stuff up so why not.